### DISCUSSION TITLE
Multi-threaded runtime shows `tokio::task::blocking`  warnings per thread/task - any tips with dealing with this?

### DISCUSSION THREAD

**Discussion by NuSkooler** [tokio-rs/console#481] [General]
Creating a multi-threaded runtime and spawning with `block_on()` results in many `tokio::task::blocking` items in `tokio-console` with warnings next to them. Any tips with dealing with these? Is it a bug that they are in warning state? Any way to filter/etc.? Code + SS below:

```rust
fn main() {
    console_subscriber::init();

    let runtime = tokio::runtime::Builder::new_multi_thread()
        .enable_all()
        .build().unwrap();


    let handle = runtime.handle();

    runtime.block_on(async move {
        let task = handle.spawn(async move {
            loop {
                tokio::time::sleep(std::time::Duration::from_secs(1)).await;
                println!("Next...");
            }
        });

        let _ = task.await;
    });
}
```

![image](https://github.com/tokio-rs/console/assets/1509339/be12a7e5-d9e1-4631-bdca-de190ccc0980)

**Comment by hds** [comment]
These tasks are the blocking task pool that Tokio itself creates. They're not async tasks, but tokio-console doesn't take that into consideration and so the warnings next to them are incorrect.

There's a fix for this being worked on and it should make it into the next version of tokio-console.

**Reply by NuSkooler** [reply]
@hds Thank you for the reply! I look forward to the next release!
